source/accounts/index.rst

```diff
@@ -324,31 +324,137 @@ RSA software tokens provide two factor authentication (2FA) for NOAA RDHPCS syst
      If you don’t have a smartphone, you can request an RSA hardware token. The activation process is found here: `New User Activation <https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/New_User_Activation#RSA_Token_Activation>`_.  RSA software tokens are preferred.
 
 
-CAC Access
-----------
+<<<<<<< Updated upstream
+Accessing the RDHPCS Systems:  CAC Access
+*****************************************
+=======
+Accessing the RDHPCS Systems
+============================
+>>>>>>> Stashed changes
 
 The Common Access Card (CAC), is the preferred means of access to RDHPCS resources for both Web and SSH access. To obtain a CAC, work with your local admin services team as they need to start the application process.  Some labs can issue CACs on-site, otherwise you will have to visit a RAPIDS site. The site locator website is `ID Card Office Online <https://idco.dmdc.osd.mil/idco/locator>`_.  SSH logins with a CAC require additional software.
 
-
-Accessing the RDHPCS Systems
-----------------------------
-
 .. NOTE::
 
    To access a system, you must be on a project assigned to that system.
 
-On-Premise RDHPCS systems (Gaea, Hera, Jet, Niagara, PPAN) are accessed via SSH.  See the following pages for detailed instructions:
+<<<<<<< Updated upstream
+**Cloud Computing**
 
-* `RSA logins <https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/RSA_Login>`_
-* `CAC logins <https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/CAC_Login>`_
+The Cloud Platform allows RDHPCS users to create a high-performance computational cluster on a cloud-based platform (AWS, Azure or GCP) with resources that are appropriate for specific processing tasks. Cloud access is mediated through the Parallel Works application. Login `here. <https://noaa.parallel.works/log>`_
 
 MSU systems (Orion, Hercules) are accessed via SSH or OpenOnDemand. See `Orion login: <https://oriondocs.rdhpcs.noaa.gov/wiki/index.php/Logging_in>`_ for detailed instructions.
+ 
```

Comment:
Oh oof.  I see what happened here.  there's two set of changes here... 
hrm..

Reply:
some of my comments above are irrelevant then.

Reply:
It's actually ok, I was working on a couple of files at the same time and crossed over my own trail. So instead of references to the wiki docs I've put internal cross references. I was going to do that last, but now's a good time.